.github/workflows/reusable-aura-tests.yml

```diff
@@ -75,6 +75,7 @@ jobs:
           yarn test:int
         working-directory: packages/${{ matrix.package }}
         env:
+          DEFAULT_DB: true
```

Comment:
Is DEFAULT_DB the best name for this? To me it's not clear it has anything to do with multi db

Reply:
Well it's not anything to do with multi DB really, it's just telling the tests to use the default DB for the DBMS - I figured this was better than `DISABLE_MULTI_DB` or something.

Reply:
Could potentially be `USE_DEFAULT_DB` then? Although it's not super important

Reply:
Sure, I've done that!!